starter-code/js/index.js

```diff
@@ -3,9 +3,31 @@ var $calc = document.getElementById('calc');
 
 function updateSubtotal($product) {
   // Iteration 1.1
+  let subtot = document.querySelector('.subtot span').innerText;
```

Comment:
This will work for the first product, but you need to also include the other products. A querySelectorAll will do the trick, or you can see what we did together in class:)